src/Field/BasicField.php

```diff
@@ -115,6 +115,12 @@ public function fetchFieldKey($fieldName)
      */
     public function fetchFieldType($fieldKey)
     {
+        // Check if this a clone field. If it is, return the field type of its clone.
+        $fieldKeyIds = explode('_', $fieldKey);
+        if (count($fieldKeyIds) == 4) {
```

Comment:
well the field name wont matter here, as this function gets the field type using the keys which are always field_{somehexvalue} e.g field_349efabab23 so the execution path is quite predictable